travel.py

```diff
@@ -45,33 +49,37 @@
 
 print("Connecting to: " + CONNSTR)
 
-app = Flask(__name__, static_url_path='/static')
+app = Flask(__name__)
+app.config.from_object(__name__)
+app.config['CORS_HEADERS'] = 'Content-Type'
 
+api = Blueprint("api", __name__)
 
-@app.route('/')
-def index():
-    return redirect("/index.html", code=302)
+CORS(app, headers=['Content-Type', 'Authorization'])
 
 
-@app.route('/<path:path>')
-def serveFiles(path):
-    return send_from_directory("static", path)
-
+@app.route('/')
+def index():
+    endpoints = []
+    for rule in app.url_map.iter_rules():
+        endpoints.append(str(rule))
```

Comment:
might be clearer as list comprehension (I was restraining myself, but I saw you use one later on line 296 so...)

Reply:
My Go brain should be switched off when I do Python :D